### ISSUE TITLE
Compatibility with VEINS tutorial

### ISSUE THREAD

**Issue by namnatulco** [sommer/veins#24]
It looks as though the most recent version of VEINS is no longer compatible with OMNeT++ 4.x, but the [tutorial](http://veins.car2x.org/tutorial/) still recommends this (old) version of OMNeT++. Are there plans to make this backwards compatible again? The cluster that I'm running my simulations on hasn't upgraded to OMNeT++5 yet, and I'd prefer not to manually compile OMNeT++ there (although I plan to check whether that is an option). In any case it'd probably be a good idea to update the tutorial.

The problem can be traced back to `veins/modules/phy/SampledAntenna1D.cc`, specifically:
```
In constructor ‘SampledAntenna1D::SampledAntenna1D(std::vector<double>&, std::string, std::vector<double>&, std::string, std::vector<double>&, cRNG*)’:
veins/modules/phy/SampledAntenna1D.cc:19:5: error: ‘cRandom’ was not declared in this scope
     cRandom* offsetGen = 0;
```

Grepping through the OMNeT++ 4.6 source code indeed does not result in any definitions of this type (and it does find the corresponding code for 5.0). 

It looks as though the corresponding errors were introduced by b8637c8 (which is the last commit that touched those files, and commits before that compile correctly). I remember that OMNeT++5 introduced some new random distribution stuff, so probably that's what's up here.

**Comment by sommer** [comment]
Thanks for catching this! Seems like I did not pay attention to backwards compatibility when merging the new antenna patterns feature. I have now fixed the tutorial and changelog to point out that Veins 4.5 needs OMNeT++ 5 to compile.

I know this is non-ideal (the release was supposed to be a minor version bump, not one that requires wholly new software), but the next version will be incompatible with OMNeT++ 4.6 anyhow.

As a quick fix for backwards compatibility, I don't have any better suggestion than patching the code to not support random variations in antenna patterns.
